ui/css/itcss/components/send.scss

```diff
@@ -1,927 +0,0 @@
-  transform: rotate(45deg);
-  left: 107px;
-  top: 294px;
-  box-shadow: var(--shadow-size-xs) var(--color-shadow-default);
-}
-
-.gas-tooltip {
-  display: flex;
-  justify-content: center;
-}
-
-.gas-tooltip-label {
-  @include design-system.Paragraph;
-
-  color: var(--color-text-alternative);
-}
-
-.gas-tooltip-header {
-  padding-bottom: 12px;
-}
-
-.gas-tooltip-input-label {
-  margin-bottom: 5px;
-}
-
-.gas-tooltip-input-label i {
-  color: var(--color-icon-muted);
-  margin-left: 6px;
-}
-
-.gas-tooltip-input-detail {
-  @include design-system.H7;
-
-  position: absolute;
-  top: 4px;
-  right: 26px;
-  color: var(--color-text-muted);
-}
-
-.gas-tooltip-input-arrows {
-  @include design-system.H6;
-
-  position: absolute;
-  top: 0;
-  right: 4px;
-  width: 17px;
-  height: 28px;
-  border: 1px solid var(--color-icon-muted);
-  border-left: 0;
-  display: flex;
-  flex-direction: column;
-  color: var(--color-text-muted);
-  padding: 1px 4px;
-  cursor: pointer;
-}
-
-.token-gas {
-  &__amount {
-    display: inline-block;
-    margin-right: 4px;
-  }
-
-  &__symbol {
-    display: inline-block;
-  }
-}
-
-.send-screen {
-  &__title {
-    @include design-system.H4;
-
-    color: var(--color-text-alternative);
-  }
-
-  &__subtitle {
-    @include design-system.H6;
-
-    margin: 10px 0 20px;
-  }
-
-  &__send-button,
-  &__cancel-button {
-    width: 163px;
-    text-align: center;
-  }
-
-  &__send-button__disabled {
-    opacity: 0.5;
-    cursor: auto;
-  }
-}
-
-.send-token {
-  display: flex;
-  flex-flow: column nowrap;
-  z-index: 25;
-
-  &__content {
-    width: 498px;
-    height: 605px;
-    background-color: var(--color-background-default);
-    box-shadow: var(--shadow-size-xs) var(--color-shadow-default);
-    padding: 46px 40.5px 26px;
-    position: relative;
-    align-items: center;
-    display: flex;
-    flex-flow: column nowrap;
-    flex: 1 0 auto;
-
-    @include design-system.screen-sm-max {
-      top: 0;
-      width: 100%;
-      box-shadow: none;
-      padding: 12px;
-    }
-  }
-
-  .identicon {
-    position: absolute;
-    top: -35px;
-    z-index: 25;
-
-    @include design-system.screen-sm-max {
-      position: relative;
-      top: 0;
-      flex: 0 0 auto;
-    }
-  }
-
-  &__title {
-    @include design-system.H4;
-
-    color: var(--color-text-alternative);
-  }
-
-  &__description,
-  &__balance-text,
-  &__token-symbol {
-    @include design-system.H6;
-
-    margin-top: 10px;
-    text-align: center;
-  }
-
-  &__token-balance {
-    @include design-system.H1;
-
-    margin-top: 13px;
-
-    .token-balance__amount {
-      padding-right: 12px;
-    }
-  }
-
-  &__button-group {
-    display: flex;
-    flex-flow: column nowrap;
-    align-items: center;
-    flex: 0 0 auto;
-
-    @include design-system.screen-sm-max {
-      margin-top: 24px;
-    }
-
-    button {
-      width: 163px;
-    }
-  }
-}
-
-.confirm-send-token {
-  &__hero-amount-wrapper {
-    width: 100%;
-  }
-}
```

Comment:
Not used as a classname

![Screenshot 2025-02-07 at 12 02 45 PM](https://github.com/user-attachments/assets/af06b78f-d0a2-4a0b-bd0c-628cc4a20865)